src/components/Article.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { fetchArticle } from "../utils/api";
+import CardHeader from "./CardHeader";
+import CardFooter from "./CardFooter";
+
+function Article() {
+  const [article, setArticle] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
+
+  const { id } = useParams();
+
+  useEffect(() => {
+    fetchArticle(id).then((article) => {
+      setArticle(article);
+      setIsLoading(false);
+    });
+  }, [id]);
+
+  if (isLoading) return <p>Getting your article...</p>;
+  else
+    return (
```

Comment:
I'm being v picky but because your if returns something then you may omit the else 😁